config.js

```diff
@@ -254,6 +258,9 @@ module.exports = {
   getSamplesWildcardCacheInvalidation:
     pe.GET_SAMPLES_WILDCARD_CACHE_INVALIDATION ||
     DEFAULT_GET_SAMPLES_WILDCARD_CACHE_INVALIDATION,
+  missedCollectorHeartbeatCheckIntervalMillis:
+    pe.MISSED_COLLECTOR_HEARTBEAT_CHECK_INTERVAL_MILLIS ||
```

Comment:
should be `+ pe.MISSED_COLLECTOR_HEARTBEAT_CHECK_INTERVAL_MILLIS` to coerce the env var (string) to number